ceph.spec.in

```diff
@@ -207,6 +207,7 @@ BuildRequires:	libtool
 BuildRequires:	libxml2-devel
 BuildRequires:	make
 BuildRequires:	ncurses-devel
+BuildRequires:	ninja-build
```

Comment:
please move this to conditional buildrequires section. On SUSE the line should be:

```
BuildRequires: ninja
```

Reply:
@smithfarm fixed and repushed.